core/src/host_env.rs

```diff
@@ -67,7 +67,7 @@ impl HostEnv {
         let use_proxy = T::cl_type() != <()>::cl_type() || !call_def.attached_value().is_zero();
         let call_result = backend.call_contract(&address, call_def, use_proxy);
 
-        let mut events_map: BTreeMap<Address, Vec<odra_types::Bytes>> = BTreeMap::new();
+        let mut events_map: BTreeMap<Address, Vec<crate::Bytes>> = BTreeMap::new();
```

Comment:
```suggestion
        let mut events_map: BTreeMap<Address, Vec<Bytes>> = BTreeMap::new();
```
already imported in L9